src/neo/SmartContract/InteropService.Storage.cs

```diff
@@ -40,7 +40,7 @@ private static long GetStoragePrice(EvaluationStack stack, StoreView snapshot)
                     Key = key.GetSpan().ToArray()
                 };
                 var skeyValue = snapshot.Storages.TryGet(skey);
-                if (skeyValue is null)
+                if (skeyValue is null || skeyValue.Value is null)
```

Comment:
It impossible for it to be `null`.

Reply:
Okay, I'll reset it. We have created a null value in UT before.